File: 2021/07/14/main.go
package main

import "strings"

func customSortString(order string, str string) string {
	res := ""
	for i := 0; i < len(order); i++ {
		o := order[i : i+1]
		count := strings.Count(str, o)
		res += strings.Repeat(o, count)
		str = strings.Replace(str, o, "", -1)
	}

	return res + str
}

File: 2021/07/14/main.py
class Solution:
    def customSortString(self, order: str, str: str) -> str:
        res = ''
        for i in range(len(order)):
            o = order[i:i+1]
            count = str.count(o)
            res += o * count
            str = str.replace(o, "", -1)
        return res + str